src/main/java/eu/yals/constants/MimeType.java

```diff
@@ -11,6 +11,7 @@ private MimeType() {
         throw new UnsupportedOperationException("Utility class");
     }
 
+    public static final String ALL = "*/*";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5392475)